registered_span.go

```diff
@@ -1054,3 +1064,53 @@ func NewAWSDynamoDBSpanTags(span *spanS) AWSDynamoDBSpanTags {
 
 	return tags
 }
+
+type AWSInvokeSpanTags struct {
+	// FunctionName is a name of the function which is invoked
+	FunctionName string `json:"function,omitempty"`
```

Comment:
Is this field mandatory? Shall we remove `omitempty` then?

Reply:
I think it makes sense.